File: Convert_Types/Decimal_Octal.c
#include <stdio.h>
void main()
{

    int decimal;

    printf("Enter a decimal number: ");
    scanf("%d", &decimal);

    int octal[20];
    int i = 0;

    while (decimal > 0)
    {
        octal[i] = (decimal % 8);
        decimal /= 8;
        i++;
    }

    printf("Octal Equivalent = ");

    for(i = i - 1; i >= 0; i--){
        printf("%d", octal[i]);
    }
}

File: Convert_Types/Decimal_Octal.java
import java.util.Scanner;
public class Decimal_Octal {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a decimal number: ");
        int decimal = sc.nextInt();

        String octal = "";

        while (decimal > 0) {
            octal = (decimal % 8) + octal;
            decimal /= 8;
        }

        System.out.println("Octal Equivalent = " + octal);
    }
}

